test/sslapitest.c

```diff
@@ -11104,6 +11104,8 @@ static size_t record_pad_cb(SSL *s, int type, size_t len, void *arg)
  * Test 1: Record padding callback on the SSL
  * Test 2: Record block padding on the SSL_CTX
  * Test 3: Record block padding on the SSL
+ * Test 4: Record block padding on the SSL_CTX with flags
+ * Test 5: Record block padding on the SSL with flags
```

Comment:
Not sure what "with flags" means here - you're testing the handshake message adding

Reply:
that's a hangover in the comment from an earlier iteration - changed to: `Test 4: Extended record block padding on the SSL_CTX` and same for 5.